liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PluginConfigSupport.java

```diff
@@ -137,7 +137,10 @@ protected File exportParametersToXml() throws Exception {
             }
         }
 
-        configDocument.createElement("appsDirectory", getAppsDirectory());
+        if (isConfigCopied()) {
+            configDocument.createElement("appsDirectory", getAppsDirectory());
+        }
+
```

Comment:
I like the general idea of using a static to keep track of the fact that copyConfigFiles() has been called.    

I think, better than checking the static flag here in  exportParametersToXml() we should check in **io.openliberty.tools.maven.server.PluginConfigSupport.getAppConfigLocationsFromSourceServerXml()**

E.g. have:
```java
    protected Set<String> getAppConfigLocationsFromSourceServerXml() {

        ServerConfigDocument scd = null;

        File serverXML = new File(serverDirectory, "server.xml");

        if (serverXML != null && serverXML.exists())  && isConfigCopied()) 
```

We go through this same method in the first create, and because the server.xml doesn't exist in the target server (which of course hasn't been created yet), we return an empty set which gets written as "dropins" in the plugin cfg XML.    

Rather than adding another branch in the place you suggested, I think a bit better to expand the conditions that we abort in **getAppConfigLocationsFromSourceServerXml()**.


(Of course, this would all be simpler if we didn't have to wait to call copyConfigFiles() until the server gets created).